src/qaqc/download_and_flag_fxns/add_frozen_flag.R

```diff
@@ -0,0 +1,20 @@
+add_frozen_flag <- function(df){
+
+  # create a df of temperature for each site
+  temperature <- df %>%
+    data.table::data.table() %>%
+    dplyr::select(DT_round, DT_join, parameter, mean) %>%
+    dplyr::filter(parameter == "Temperature") %>%
+    dplyr::select(DT_join, Temperature = mean)
+
+    # add "temperature" column to df:
```

Comment:
This could be a good method for adding the other metadata parameters to the data frames that we have.

Reply:
Yes!